src/picterra/client.py

```diff
@@ -943,6 +943,32 @@ def delete_vector_layer(self, vector_layer_id: UUID):
         if not resp.ok:
             raise APIError(resp.text)
 
+    def download_vector_layer_to_file(self, vector_layer_id: UUID, filename: str):
+        """
+        Downloads a vector layer
+
+        This a **beta** function, subject to change.
+
+        Args:
+            vector_layer_id: The id of the vector layer to remove
```

Comment:
left-over from copy-pasting? We are not removing anything here, right?

Reply:
fixed